src/ocamlorg_frontend/pages/community.eml

```diff
@@ -37,27 +37,35 @@ Layout.render
         </a>
         <a href="https://github.com/ocaml/ocaml" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/github.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Github</div>
-          <div class="font-normal text-xs mt-4">Open bug reports and feature requests on the compiler</div>
+          <div class="font-normal text-xs mt-4">Open bug reports and feature requests on the OCaml compiler.</div>
         </a>
         <a href="https://discord.gg/cCYQbqN" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/discord.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Discord</div>
           <div class="font-normal text-xs mt-4">Chat with other members of the community on Discord.</div>
         </a>
         <a href="https://twitter.com/ocamllang" class="border-gray-200 border p-8 rounded-xl card-hover">
           <div class="text-base font-semibold"><img src="/img/community/twitter.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Twitter</div>
-          <div class="font-normal text-xs mt-4">Catch up with some of the latest announcement and events from the community.</div>
+          <div class="font-normal text-xs mt-4">Catch up with some of the latest announcements and events from the community.</div>
+        </a>
+        <a href="https://types.pl/" class="border-gray-200 border p-8 rounded-xl card-hover">
+          <div class="text-base font-semibold"><img src="/img/community/mastodon.svg" alt="" class="h-8 w-8 mr-2 inline-block" /> Mastodon</div>
+          <div class="font-normal text-xs mt-4">Join the community on Mastodon, a decentralized social network. The types.pl Mastodon homeserver is popular among the OCaml community.</div>
```

Comment:
```suggestion
          <div class="font-normal text-xs mt-4">Join the community on Mastodon, a decentralized social network. The <a href="https://types.pl/">types.pl</a> or <a href="https://discuss.systems/">discuss.systems</a> Mastodon homeservers are both popular choices among the OCaml community.</div>
```

Reply:
We're already in a HTML `<a>`-tag here that links to something, so putting two links inside that won't work (the small links will be super small and hard to click within the huge box which is already a link). So, either we can link to Mastodon in general, and then mention these servers, or choose one to primarily link to and mention the other in the text.

Reply:
That or have two Mastodon boxes, one for each server.

Reply:
not making them linkable should be fine, but I think it's important to mention two possible servers.